docs/source/guides/integrations.mdx

```diff
@@ -0,0 +1,279 @@
+# Integrate any ML framework with the Hub
+
+The Hugging Face Hub makes hosting and sharing models with the community easy. It supports
+[dozens of libraries](https://huggingface.co/docs/hub/models-libraries) in the Open Source ecosystem. We are always
+working on expanding this support to push collaborative Machine Learning forward. The `huggingface_hub` library plays a
+key role in this process, allowing any Python script to easily push and load files.
+
+There are four main ways to integrate a library with the Hub:
+1. **Push to Hub:** implement a method to upload a model to the Hub. This includes the model weights, as well as
+   [the model card](https://huggingface.co/docs/huggingface_hub/how-to-model-cards) and any other relevant information
+   or data necessary to run the model (for example, training logs). This method is often called `push_to_hub()`.
+2. **Download from Hub:** implement a method to load a model from the Hub. The method should download the model
+   configuration/weights and load the model. This method is often called `from_pretrained` or `load_from_hub()`.
+3. **Inference API:** use our servers to run inference on models supported by your library for free.
+4. **Widgets:** display a widget on the landing page of your models on the Hub. It allows users to quickly try a model
+   from the browser.
+
+In this guide, we will focus on the first two topics. If you are interested in Inference and Widgets, you can follow
+[this guide](https://huggingface.co/docs/hub/models-adding-libraries#set-up-the-inference-api). In both cases, you can
+reach out to us if you are integrating your library with the Hub and want to be listed [in our docs](https://huggingface.co/docs/hub/models-libraries). 
+
+## A first approach: helpers
```

Comment:
A bit nitpicky, but I'd consider naming this section something like "A flexible approach: helpers"?. The "first approach" suggests users should try this first, which makes it seem like it's the recommended way to integrate (unless this is the recommended way 😛).

Reply:
Yep, good point :+1: